plugins/modules/lvg.py

```diff
@@ -418,89 +369,40 @@ def make_vg(module, vg_name, vg_state):
     global result
 
     pvs = module.params['pvs']
-
-    cmd = "mkvg "
+    opt = build_vg_opts(module)
 
     vg_type_opt = {
         "none": '',
         "big": '-B ',
         "scalable": '-S ',
     }
```

Comment:
Should we add small here? The "small" vg type was added to main argument_spec.

Reply:
no, we should not be adding small here. however you are right. i forgot to cleanup the argument spec. i will be removing 'small' in there. thanks